packages/xeus/src/web.worker.kernel.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Thorsten Beier
+// Copyright (c) JupyterLite Contributors
+// Distributed under the terms of the Modified BSD License.
+
+import coincident from 'coincident';
+
+import { wrap } from 'comlink';
+import type { Remote } from 'comlink';
+
+import { PromiseDelegate } from '@lumino/coreutils';
+
+import { KernelMessage } from '@jupyterlab/services';
+
+import {
+  DriveContentsProcessor,
+  TDriveMethod,
+  TDriveRequest
+} from '@jupyterlite/contents';
+
+import { WebWorkerKernelBase } from '@jupyterlite/xeus-core';
+import { IEmpackXeusWorkerKernel } from './interfaces';
+import { PageConfig } from '@jupyterlab/coreutils';
+
+export class WebWorkerKernel extends WebWorkerKernelBase {
+  /**
+   * Instantiate a new WebWorkerKernel
+   *
+   * @param options The instantiation options for a new WebWorkerKernel
+   */
+  constructor(options: WebWorkerKernel.IOptions) {
+    super(options);
+  }
```

Comment:
```suggestion
```

Reply:
We need to keep this so the type of the options is correct from the outside (we don't define an interface for this class, maybe we should)